docs/users/included-tools.md

```diff
@@ -0,0 +1,19 @@
+<h1>Developer Tools Included in the Container</h1>
+We have included several useful tools for Developers in our containers.
+
+### Command-line Tools
+- [Composer](https://getcomposer.org/) - Dependency Manager for PHP
+- [Drush](http://www.drush.org) - Command-line shell and Unix scripting interface for Drupal.
+- [WP-CLI](http://wp-cli.org/) - Command-line tools for managing WordPress installations.
+
+### Email
+[MailHog](https://github.com/mailhog/MailHog) is a mail catcher we have installed and configured to catch emails sent by PHP.
```

Comment:
I don't think the "we" everywhere is completely necessary. Maybe "Mailhog is a mail catcher which is configured to capture and display emails sent by the PHP in the development site"